packages/ext-angular-modern-kitchensink/src/app/view/landingpage/landingpage.component.html

```diff
@@ -45,22 +45,43 @@
   
   
       <!-- Breadcrums starts-->
-      <!-- docked="top" node={selectedNavNode} -->
-  
+
       <toolbar #item [docked]="'top'">
-        <button #item 
-        [text]="nodeText" 
-        [ui]="app-breadcrumb"
-        [handler]="navigate"
-        ></button>
-      </toolbar>  
+        <ng-template ngFor let-segment [ngForOf]="breadcrumb">
+          <button #item
+          [text]="segment.text"
+          [cls]="'breadcrumb-button'"
+          (tap)="navigate(segment.path)"
+          >
+          </button>
+          <button #item *ngIf="!segment.isLeaf"
```

Comment:
Does not seem to be correct. Let's say user clicks on component. It is not a leaf so it will add a ">" in front of it though it is the end.

Reply:
I do need to clean up the breadcrumb a bit still.  This is a work in progress.